flybywiresim/a32nx
# feat: opt-in sentry.io error reporting

## Summary of Changes


This adds **opt-in**, **anonymous** error reporting via sentry.io.

![image](https://user-images.githubusercontent.com/4503241/153697796-84d85091-9100-4121-93bb-15e399e15e93.png)

* [x] Add sentry.io integration client
* [x] Add consent popup
* [x] Add EFB toggle
* [x] Add initialization code for leagcy JS gauges
* [x] Add initialization code for TS gauges

This is enabled on all production builds. This will not run on local or PR builds.

Clients are instantiated on all instruments, and a "root" client on the EFB performs the consent gathering flow.

## Testing instructions


* Make sure you get the consent popup on loading the aircraft and that you do not subsequently get it again
* Make sure the EFB toggle works
* Contact a dev team member to test if the consent is being handled properly


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
